commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt

```diff
@@ -19,3 +24,17 @@ class RegisterRequest(
     @JsonProperty("recaptcha2-response")
     val captcha: String? = null
 )
+
+class CheckUserRequest(
+    @JsonDeserialize(using = EmailDeserializer::class)
+    @field:Email(message = "The provided email is not valid")
+    val userEmail: String,
+
+    val userName: String,
+)
+
+class EmailDeserializer : StdDeserializer<String>(String::class.java) {
+    override fun deserialize(p: JsonParser?, ctxt: DeserializationContext?): String {
+        return _parseString(p, ctxt).lowercase()
```

Comment:
not sure about adding case insensitve check in web layer? should we add this servie layer directly? (so if we change how we call this, rule still in place)